frontend/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Build static files
 # Node Bullseye has npm
-FROM node:20.18.1-bullseye-slim AS build
+FROM node:20-bullseye-slim AS build
```

Comment:
Please use full semvers. This isn't much better than a latest tag.

Reply:
added back minor tag